README.md

```diff
@@ -1,25 +1,37 @@
 # ksicamp-the-game
 
-## How to build
+## Building the game
+
 ### Option 1: Compile by hand
```

Comment:
Using `make` should be the preferred option and therefore number 1.

Reply:
`Done!`